scripts/globals/chocobo_raising.lua

```diff
@@ -572,6 +572,48 @@ local compareTables = function(t1, t2)
     return true
 end
 
+local condenseEvents = function(player, chocoState, events)
+    local cutEvent = function(t, eStart, eEnd, csList)
+        table.insert(t, { eStart, eEnd, csList })
+    end
+
+    local condensedEvents = {}
+    local currentStartDay = nil
+    local currentEndDay = nil
+    local currentEventCSTable = nil
+
+    -- Each event is a table of cs's
+    debug(player, "Raw Events")
+    for idx, entry in pairs(events) do
+        debug(player, "Day", entry[1], ":", entry[2][1])
+        -- Only condense days with the same table contents
+        if compareTables(entry[2], currentEventCSTable) then
+            -- Increase the span
+            currentEndDay = currentEndDay + 1
+        else
+            -- If there is an active span, cut it now
+            if currentEventCSTable then
+                cutEvent(condensedEvents, currentStartDay, currentEndDay, currentEventCSTable)
+            end
+
+            -- Start a new span
+            currentEventCSTable = entry[2]
+            currentStartDay = chocoState.report.day_start + idx - 1
```

Comment:
Just so it doesn't get lost, changing currentStartDay and currentEndDay assignments to `entry[1]` will provide the expected output.